.github/workflows/reusable-e2e.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         id: check_changes
         run: |
           CHANGED_FILES=${{ steps.get-changed-files.outputs.all }}
-          if echo "$CHANGED_FILES" | grep -q "circuit" && ! echo "$CHANGED_FILES" | grep -q ".github/workflows/circuit-build.yml"; then
+          if echo "$CHANGED_FILES" | grep -q "\.circom"; then
```

Comment:
What's this change about?

Reply:
Oh yea sorry for not mentioning in the description! Basically this change is so that the e2e test only run when the actual circom files change, not any files in the circuits folder as it was until this commit. I was planning to introduce this with another PR but got carried away by not wanting to wait on circuits compilation for this PR